README.md

```diff
@@ -1,83 +1,64 @@
 
-```bash
-make docker-serve
-```
+## SIG Docs open meetings
 
-Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+Learn how to engage with the SIG Docs Kubernetes community on the [community page](https://github.com/kubernetes/community/tree/master/sig-docs#meetings).
 
-## Running the website locally using Hugo
+You can also reach the maintainers of this project at:
 
-See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L9) file.
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
 
-To run the website locally when you have Hugo installed:
+## Contributing to the docs
 
-```bash
-make serve
-```
+You can click the **Fork** button in the upper-right area of the screen to create a copy of this repository in your GitHub account. This copy is called a *fork*. Make any changes you want in your fork, and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
 
-This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+Once your pull request is created, a Kubernetes reviewer will take responsibility for providing clear, actionable feedback.  As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**
 
-## Community, discussion, contribution, and support
+Also, note that you may end up having more than one Kubernetes reviewer provide you feedback or you may end up getting feedback from a Kubernetes reviewer that is different than the one initially assigned to provide you feedback.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+Furthermore, in some cases, one of your reviewers might ask for a technical review from a Kubernetes tech reviewer when needed.  Reviewers will do their best to provide feedback in a timely fashion but response time can vary based on circumstances.
 
-You can reach the maintainers of this project at:
+For more information about contributing to the Kubernetes documentation, see:
 
-- [Slack](https://kubernetes.slack.com/messages/sig-docs)
-- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+* [Start contributing](https://kubernetes.io/docs/contribute/start/)
+* [Using Page Templates](https://kubernetes.io/docs/contribute/style/page-templates/)
```

Comment:
nit: These links were recently renamed.

Reply:
Updated the first link to `[Contribute to Kubernetes docs](https://kubernetes.io/docs/contribute/)` do you think the others needed to be changed too?

Reply:
I think the other pages are still there.
The `Documentation Style Guide` link could use `https`?

Reply:
Otherwise, LGTM